### DISCUSSION TITLE
No longer able to define global plugin configs since updating to @quasar/vite-plugin v1.4.x

### DISCUSSION THREAD

**Discussion by ghiscoding** [quasarframework/quasar#16045] [General - Components / Directives / etc] [answered]
I have global configs defined for Quasar and that used to work properly before updating (mostly because the `config` property of `QuasarPluginOptions` was set to `any` so we could provide anything we wanted really)

So considering this global config

```ts
const quasarOptions: Partial<QuasarPluginOptions> = {
  // Quasar plugins
  plugins: {
    Dialog,
    Notify,
  },
  config: {
    /* Notify defaults */
    notify: {
      position: 'top',
      timeout: 2500,
    },
  },
};

const app = createApp(App);
app.use(Quasar, quasarOptions)
```
this no longer work and tells me what `notify` is not a known property of `QuasarPluginOptions` and seem to be caused by this [PR #15945](https://github.com/quasarframework/quasar/pull/15945)

> Type '{ notify: { position: string; timeout: number; textColor: string; actions: { icon: string; color: string; }[]; }; loading: { spinner: ComponentConstructor<QSpinnerClock>; ... 4 more ...; customClass: string; }; }' is not assignable to type 'QuasarUIConfiguration'.
  Object literal may only specify known properties, and 'notify' does not exist in type 'QuasarUIConfiguration'.ts(2322)
plugin.d.ts(13, 3): The expected type comes from property 'config' which is declared here on type 'Partial<QuasarPluginOptions>'

![image](https://github.com/quasarframework/quasar/assets/643976/2a54b937-a58a-43f7-8e19-75daa5bd97b7)

I am not expecting any errors since even the documentation still mentions the same way to configure the global configs

https://quasar.dev/quasar-plugins/notify#installation

![image](https://github.com/quasarframework/quasar/assets/643976/52812492-3cd7-48f6-837d-c97bc630b52d)

Am I doing something wrong or is that a Regression Bug?

**Comment by yusufkandemir** [comment, marked as answer]
It's a bug, thanks for reporting it. I converted this into an issue: https://github.com/quasarframework/quasar/issues/16046

The fix will be included in the next `quasar` release. It's not related to the Vite plugin.

**Reply by ghiscoding** [reply]
great thanks a lot for the info :)
